alchemist/buildSrc/src/main/kotlin/Libs.kt

```diff
@@ -359,6 +372,11 @@ object Libs {
 
     const val jool_java_8: String = "org.jooq:jool-java-8:" + Versions.jool_java_8
 
+    /**
+     * http://github.com/ronmamo/reflections
+     */
+    const val reflections: String = "org.reflections:reflections:" + Versions.reflections
```

Comment:
This does not work with Java11. Remove it, use the utility class provided in baselib